### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by jrmuizel** [d-velopment/macClick#1]
It would be nice if there was a description of how this program fixes the problem in the README

**Comment by marslicy** [comment]
I think it just plays a no sound audio on the background to prevent the sound card from sleeping

**Comment by d-velopment** [comment]
> I think it just plays a no sound audio on the background to prevent the sound card from sleeping

Correct. The app is quite simple, just preventing audio card power from switch off (after 1 sec of silence usually).

**Comment by 7s3** [comment]
> 

Does this have any effect on battery life? Sorry if it's a dumb question :)
